<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="Tokyos-Air" failures="1" tests="1" name="Default test" time="6.526" errors="0" timestamp="2021-11-22T11:56:39 SAST">
  <testcase classname="runner.ProductRunner" name="Product_1977_Rates" time="6.055">
    <failure type="org.json.JSONException" message="JSONArray[5] not found.">
      <![CDATA[org.json.JSONException: JSONArray[5] not found.
at org.json.JSONArray.get(JSONArray.java:264)
at org.json.JSONArray.getJSONObject(JSONArray.java:477)
at com.exergyRestApi.TestCases.ProductRates.Product_1977_TestCases.Product_1977(Product_1977_TestCases.java:371)
at runner.ProductRunner.Product_1977_Rates(ProductRunner.java:94)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Product_1977_Rates -->
  <testcase classname="runner.ProductRunner" name="TC_ProductRules" time="0.0">
    <ignored/>
  </testcase> <!-- TC_ProductRules -->
  <testcase classname="runner.ProductRunner" name="Product_1960E_Rates" time="0.0">
    <ignored/>
  </testcase> <!-- Product_1960E_Rates -->
  <testcase classname="runner.ProductRunner" name="Product_1960C_Rates" time="0.0">
    <ignored/>
  </testcase> <!-- Product_1960C_Rates -->
  <testcase classname="runner.ProductRunner" name="Product_60200000_Rates" time="0.0">
    <ignored/>
  </testcase> <!-- Product_60200000_Rates -->
  <testcase classname="runner.ProductRunner" name="Product_1960L_Rates" time="0.0">
    <ignored/>
  </testcase> <!-- Product_1960L_Rates -->
  <testcase classname="runner.ProductRunner" name="Product_60100000_Rates" time="0.0">
    <ignored/>
  </testcase> <!-- Product_60100000_Rates -->
</testsuite> <!-- Default test -->
